src/draw-page.scss

```diff
@@ -0,0 +1 @@
+@import url("https://rsms.me/inter/inter.css");
```

Comment:
We shouldn't need to use a custom font. We want to use the same font (IBM Plex Sans) across the application for consistency.